### DISCUSSION TITLE
Add "Remove vertex snapping" to 117 HD

### DISCUSSION THREAD

**Discussion by Zakawer** [runelite/runelite#18195] [Ideas]
Currently, only the other GPU renderer plugin (simply called "GPU") has an option to remove vertex snapping, which makes most in-game models look a lot smoother and more coherent-looking when animation smoothing is disabled.

However, I think 117 HD should also have this feature implemented as well. I'd appreciate it!

**Comment by cdfisher** [comment]
We don't maintain 117. You can make this suggestion to the 117 team [here](https://github.com/117HD/RLHD/discussions).
